.github/workflows/pluto-workflow-kustomize.yaml

```
@@ -28,5 +40,16 @@ jobs:
           UNENCRYPTED_TARGET_VERSION: ${{ secrets.TARGET_K8S_VERSION }}
           UNENCRYPTED_OPSLEVEL_PLUTO_INTEGRATION_WEBHOOK_URL: ${{ secrets.OPSLEVEL_PLUTO_INTEGRATION_WEBHOOK_URL }}
         run: |
-          kubectl kustomize ${{ inputs.template_dir }} | pluto detect - --target-versions k8s=$UNENCRYPTED_TARGET_VERSION --ignore-deprecations --ignore-removals --output json | curl -i -s -X POST $UNENCRYPTED_OPSLEVEL_PLUTO_INTEGRATION_WEBHOOK_URL?alias=${{ inputs.service_name }} -H 'content-type: application/json' --data-binary @-
+          MANIFEST_FILE=manifest.yaml
+
+          # Generate a manifest file
+          ## Kustomize
+          if [[ -n "${{ inputs.template_dir }}" ]]; then
+            kubectl kustomize ${{ inputs.template_dir }} --output=$MANIFEST_FILE
+
+          ## Helm
+          else
+            helm template charts/${{ inputs.service_name }} --values ${{ inputs.values_base }} --values ${{ inputs.values_overlay }} > $MANIFEST_FILE
+          fi
 
+          cat $MANIFEST_FILE | pluto detect - --target-versions k8s=$UNENCRYPTED_TARGET_VERSION --ignore-deprecations --ignore-removals --output json | curl -i -s -X POST $UNENCRYPTED_OPSLEVEL_PLUTO_INTEGRATION_WEBHOOK_URL?alias=${{ inputs.service_name }} -H 'content-type: application/json' --data-binary @-
```

- Quoting.
- `cat` is unnecessary in this pipeline.

```suggestion
          pluto detect - "--target-versions k8s=$UNENCRYPTED_TARGET_VERSION" --ignore-deprecations --ignore-removals --output json < "$MANIFEST_FILE" | curl -i -s -X POST "$UNENCRYPTED_OPSLEVEL_PLUTO_INTEGRATION_WEBHOOK_URL?alias=${{ inputs.service_name }}" -H 'content-type: application/json' --data-binary @-
```